central/main.go

```diff
@@ -673,13 +674,13 @@ func customRoutes() (customRoutes []routes.CustomRoute) {
 		customRoutes = append(customRoutes, routes.CustomRoute{
 			Route:         "/db/backup",
 			Authorizer:    dbAuthz.DBReadAccessAuthorizer(),
-			ServerHandler: notImplementedOnManagedServices(globaldbHandlers.BackupDB(nil, nil, globaldb.GetPostgres(), false)),
+			ServerHandler: httputil.HandlerNotImplementedIfSettingDisabled(!env.ManagedCentral.BooleanSetting(), globaldbHandlers.BackupDB(nil, nil, globaldb.GetPostgres(), false)),
```

Comment:
I don't have a good solution to that problem, but I find it kind of annoying that using the handler to convey a "not implemented" means the entire rest of the pipeline still runs - authn (probably desired), audit (maybe okay), authz (meh - it's odd to say "not authorized" on an unimplemented API). I wondered if it wouldn't be better to only conditionally append to `customRoutes`, but then we'd just get a 404 instead of a more meaningful not implemented message..